plugins/net_plugin/net_plugin.cpp

```diff
@@ -3532,6 +3576,8 @@ namespace eosio {
            "   _port  \tremote port number of peer\n\n"
            "   _lip   \tlocal IP address connected to peer\n\n"
            "   _lport \tlocal port number connected to peer\n\n")
+         ( "keepalive-interval", bpo::value<int>()->default_value(def_keepalive_interval), "heartbeat keepalive message interval in miliseconds")
```

Comment:
This should be called `p2p-keepalive-interval-ms`
Remember to update the PR description.